lrpar/src/lib/ctbuilder.rs

```diff
@@ -398,11 +402,19 @@ where
                         };
                     }
                     if let Some(s) = grm.action(pidx) {
-                        let ns = re.replace_all(
-                            s,
-                            format!("{prefix}arg_$1", prefix = ACTION_PREFIX).as_str()
+                        // Replace $1 ... $n with the correct local variable
```

Comment:
This will also overwrite any `$X`s inside of strings which can restrict the user from some reasonable actions. Probably something we'll have to address in the future.

Reply:
Agreed, but fixing this perfectly would be incredibly hard -- close to impossible (we'd have to parse arbitrary Rust fragments, which is hard, given that Rust doesn't have a formal grammar). I think we might just have to document this as "gotcha" somewhere?

Reply:
I quickly checked, and Yacc ignores `$X` within strings, so we should at least document this somewhere. My first thought at a somewhat ok fix for this would be to count unescaped quotes and only replace `$X` if the count is even. It's not enough, but it might be better than nothing?

Reply:
Agreed, we could do that. Do you want to file an issue and tackle it in another PR? I don't really want to make this PR any more complex than it already is!

Reply:
Sounds like a plan!